test/README.md

```diff
@@ -6,35 +6,81 @@ Thanks for writing tests! Here's a quick run-down on our current setup.
 
 Please familiarise yourself with these if you plan on contributing! :+1:
 
- - [enzyme](https://github.com/airbnb/enzyme)
- - [mocha](https://mochajs.org)
- - [chai](http://chaijs.com)
- - [jsdom](https://github.com/tmpvar/jsdom)
-
+- [mocha](https://github.com/mochajs/mocha)
+- [enzyme](https://github.com/airbnb/enzyme)
+- [jsdom](https://github.com/tmpvar/jsdom)
+- [karma](https://github.com/karma-runner/karma)
+- [docker](https://github.com/docker/docker)
+- [chai](https://github.com/chaijs/chai)
+- [sinon](https://github.com/sinonjs/sinon)
 
 ## Commands
 
-##### Run the core mocha unit/integration test suite.
+Material-UI uses a wide range of tests approach as each of them comes with a different
+tradeoff. It's mainly a completeness/speed tradeoff.
+
+### React API level
+
+First, we have the **unit test** suite.
+It's using [mocha](https://mochajs.org) and the *shallow* API of [enzyme](https://github.com/airbnb/enzyme).
+It's allowing testing the components in isolation.
```

Comment:
It uses [mocha](https://mochajs.org) and the *shallow* API of [enzyme](https://github.com/airbnb/enzyme) to allow testing the components in isolation.